config/config.go

```diff
@@ -211,6 +213,7 @@ func createDefaultConfig() *Config {
 		FileOpenHandlers:                            make(map[string]string),
 		FileOpenSaveFilesPermanently:                false,
 		FileDownloadSaveLocation:                    "~/Downloads",
+		VimEnabled: false,
```

Comment:
Please format the whole codebase with `gofmt -w -s .` And always use a formatter in your editor please if you forget to do this before the commit.

Reply:
That indentation was correctly formatted some commits ago. I don't know what broke it